puppet/modules/jenkins_job_builder/files/centos.org/ansible/jenkins_job.yml

```diff
@@ -99,7 +99,7 @@
         password: "{{ jenkins_password }}"
         url: "{{ jenkins_host }}/job/{{ jenkins_job_name }}/{{ poll_result.json.executable.number }}/api/json"
       register: build_job_result
-      until: (build_job_result.json is defined) and (build_job_result.json.result == 'SUCCESS' or build_job_result.json.result == 'FAILURE')
+      until: (build_job_result.json is defined) and (build_job_result.json.result == 'SUCCESS' or build_job_result.json.result == 'FAILURE') and (not build_job_result.json.building)
```

Comment:
In my logic I'd expect the building check before the result check, but it probably doesn't matter.

Reply:
hah, I thought that too after submiting the PR, reordered.